lib/widgets/features/explore/screens/explore.dart

```diff
@@ -148,10 +148,11 @@ class _ExploreCardScrollState extends State<ExploreCardScroll> {
           (user) => createProfileCardFromInfo(
             info: ProfileQuickViewInfo(
               userId: user.id,
-              userType:
-                  user.offersHelp ? UserType.mentor : UserType.entrepreneur,
+              userType: user.offersHelp == true
+                  ? UserType.mentor
+                  : UserType.entrepreneur,
               avatarUrl: user.avatarUrl,
-              fullName: user.fullName!,
+              fullName: user.fullName ?? '${user.firstName} ${user.lastName}',
```

Comment:
Is there a way to have this logic in the backend so that instead of storing fullName as a separate field it would instead be calculated from the firstName and lastName?